docs/content/posts/2024-06-14-release-1.19.1.md

```diff
@@ -0,0 +1,171 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-34725'>FLINK-34725</a>] -         Dockerfiles for release publishing has incorrect config.yaml path
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-34933'>FLINK-34933</a>] -         JobMasterServiceLeadershipRunnerTest#testResultFutureCompletionOfOutdatedLeaderIsIgnored isn&#39;t implemented properly
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-34956'>FLINK-34956</a>] -         The config type is wrong for Duration
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35000'>FLINK-35000</a>] -         PullRequest template doesn&#39;t use the correct format to refer to the testing code convention
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35089'>FLINK-35089</a>] -         Two input AbstractStreamOperator may throw NPE when receiving RecordAttributes
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35097'>FLINK-35097</a>] -         Table API Filesystem connector with &#39;raw&#39; format repeats last line
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35098'>FLINK-35098</a>] -         Incorrect results for queries like &quot;10 &gt;= y&quot; on tables using Filesystem connector and Orc format
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35112'>FLINK-35112</a>] -         Membership for Row class does not include field names
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35159'>FLINK-35159</a>] -         CreatingExecutionGraph can leak CheckpointCoordinator and cause JM crash
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35169'>FLINK-35169</a>] -         Recycle buffers to freeSegments before releasing data buffer for sort accumulator
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35217'>FLINK-35217</a>] -         Missing fsync in FileSystemCheckpointStorage
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35351'>FLINK-35351</a>] -         Restore from unaligned checkpoints with a custom partitioner fails.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35358'>FLINK-35358</a>] -         Breaking change when loading artifacts
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35383'>FLINK-35383</a>] -         Update compatibility matrix to include 1.19 release
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35429'>FLINK-35429</a>] -         We don&#39;t need introduce getFlinkConfigurationOptions for SqlGatewayRestEndpointFactory#Context
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35526'>FLINK-35526</a>] -         Remove deprecated stedolan/jq Docker image from Flink e2e tests
+</li>
+</ul>
+
+<h2>        New Feature
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-34633'>FLINK-34633</a>] -         Support unnesting array constants
```

Comment:
In general, the patch version shouldn't include the new Feature.

After I checked, i think the `Fix Version` of FLINK-34633 should be marked as `1.20.0` instead of `1.19.1`. Let's follow it in FLINK-34633.

Reply:
Removed. Thank you for checking!